model/src/main/proto/format_import_proto_library.bzl

```diff
@@ -1,9 +1,23 @@
 
 def format_import_proto_library(name, src, deps):
-  """
-  Formats the import statement of the src file to contain a full path to the file
-  The genrule() rule generates a copy of the src file and alters the copy
-  The proto_library() rule takes this altered copy and builds it
+  """This macro exists as a way to build proto files that contain import statements in both Gradle
+  and Bazel.
+  Currently the files that use this macro are: exploration.proto, topic.proto, and question.proto.
+  This macro formats the src file's import statements to contain a full path to the file in order
+  for Bazel to properly locate file.
+  The genrule() rule generates an altered copy of the src file for Bazel to build, leaving the
```

Comment:
Everything above this is great for method documentation. Everything below it seems more appropriate as inline comments since it’s explaining the implementation. Method documentation is for API, not implementation. Users don’t care how a method does what it does, they care about what it actually intends to do.

Reply:
okay! fixed!